packages/xrpc-server/src/util.ts

```diff
@@ -268,10 +268,6 @@ function decodeBodyStream(
   return stream
 }
 
-export const getReqIp = (req: express.Request): string => {
-  return req.ips.at(-1) ?? req.ip
```

Comment:
Part of the issue here is that `req.ips` was empty since `'trust proxy'` was `false`, so fell back to `req.ip`, which was the raw socket address (again since `'trust proxy'` was `false`).

Reply:
yup totally 👍 

altho this would still need to grab the first ip in the array i believe (`req.at(0)`) which is exactly what `req.ip` does so i figured i'd just simplify it